Exercise_B/index.html

```diff
@@ -15,20 +15,78 @@
     />
     <!-- Custom Stylesheet -->
     <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
   </head>
 
   <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+    <nav id="icon"><img src="./assets/images/logo.png" alt="Italian Trulli">
+    </nav>
 
+    <div class="flower1">
+      <p class="bea">Beautiful Flowers</p>
+      <br>
+      <p class="learn">Learn more</p>
+    </div>
     <!--Four containers with img / text-->
     <!-- 4 containers -->
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-6 col-lg-3 col-sm-12">
+          <div class="row">
+            <div class="col-md-6 col-sm-6">
+          <img src="./assets/images/flower-icon.png" alt="flowericon">
+            </div>  
+            <div class="col-md-6 col-sm-6 row align-items-center">
+            Flowers for every kind of situation
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-3 col-sm-12">
+          <div class="row">
+            <div class="col-md-6 col-sm-6">
+             <img src="./assets/images/payment-icon.png" alt="payment">
+            </div>  
+            <div class="col-md-6 col-sm-6 row align-items-center">
+             All major payments kinds accepted
+            </div>
+          </div>
+         </div>
+
+         <div class="col-md-6 col-lg-3 col-sm-12">
+          <div class="row">
+            <div class="col-md-6 col-sm-6">
+          <img src="./assets/images/phone-icon.png" alt="phone">
+            </div>  
+            <div class="col-md-6 col-sm-6 row align-items-center">
+            Free consulation
+            </div>
+          </div>
+        </div>
+        
+        <div class="col-md-6 col-lg-3 col-sm-12">
+          <div class="row">
+            <div class="col-md-6 col-sm-6">
+          <img src="./assets/images/truck-icon.png" alt="truck">
+            </div>  
+            <div class="col-md-6 col-sm-6 row align-items-center">
+            City wide delivery
+            </div>
+          </div>
+        </div>
+        
+
+
+      
+      </div>
+    </div>
 
     <!-- Payments Div -->
     <!-- Consultation Div -->
 
     <!-- Delivery Div -->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
 
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
     <!--Scripts required for bootstrap CDN-->
   </body>
 </html>

```

Comment:
Great Job, This works well!! Though you can increase the padding inside the Learn More Button